### PR TITLE
Don't show hard-to-use IDPs for Smart Lock sign-in

### PR DESCRIPTION
**Pull request by SUPERCILEX**
See #1154

### CODE REVIEW

File: auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java [hunk 1]
```diff
@@ -224,9 +222,7 @@ public void finish(int resultCode, Intent resultIntent) {
         List<String> accounts = new ArrayList<>();
         for (AuthUI.IdpConfig idpConfig : getFlowParams().providerInfo) {
             @AuthUI.SupportedProvider String providerId = idpConfig.getProviderId();
-            if (providerId.equals(GoogleAuthProvider.PROVIDER_ID)
-                    || providerId.equals(FacebookAuthProvider.PROVIDER_ID)
-                    || providerId.equals(TwitterAuthProvider.PROVIDER_ID)) {
+            if (providerId.equals(GoogleAuthProvider.PROVIDER_ID)) {
```
Comment (samtstern) [inline auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java]: nit: can we change the name of this method to be "getCredentialAccountTypes" or something that implies it has to do with SmartLock?

**Comment by samtstern** [comment]
@SUPERCILEX thanks for this!  I guess this would make the next release `3.3.0` since it's a behavior change?

File: auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java (hunk 1 above)
Comment (SUPERCILEX) [reply auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java]: 👍

**Comment by SUPERCILEX** [comment]
Yup, SGTM

**Comment by samtstern** [comment]
@SUPERCILEX I am actually going to bump this off of `3.2.2` and into the next release since I think we have some other low-hanging minor-version improvements that it would be nice to bundle together.

This way we can keep this bug-fix release as a true bug-fix release.

**Comment by SUPERCILEX** [comment]
Yeah, SGTM. 👍

**Comment by SUPERCILEX** [comment]
@samtstern Is this ready to roll?

**Comment by samtstern** [comment]
@SUPERCILEX kept forgetting about this one, but good to go.

**Pull request merged**